src/main/webapp/components/basic-search/basic-search.js

```diff
@@ -233,26 +233,42 @@ const defaultFilters = {
   [LOCATION_KEY]: makeDefaultSearchGeo(),
 }
 
-const getFilterMap = props => {
-  if (props.query && props.query.filterTree) {
-    return fromFilterTree(props.query.filterTree, {
-      basicSearchMatchType: props.basicSearchMatchType,
+const getFilterMap = ({ query, basicSearchMatchType }) => {
+  if (!query) {
+    return Map({ text: '*', sources: undefined })
+  }
+
+  let filterMap = query.filterTree
+    ? fromFilterTree(query.filterTree, {
+        basicSearchMatchType: basicSearchMatchType,
```

Comment:
:pencil2: could use shorthand here 
```suggestion
        basicSearchMatchType,
```